signac/contrib/migration/v1_to_v2.py

```diff
@@ -50,3 +50,13 @@ def _migrate_v1_to_v2(root_directory):
     v2_fn = _get_project_config_fn(root_directory)
     os.mkdir(os.path.dirname(v2_fn))
     os.rename(v1_fn, v2_fn)
+
+    # Now move all other files.
+    files_to_move = {
+        ".signac_shell_history": os.sep.join((".signac", "shell_history")),
+        ".signac_sp_cache.json.gz": os.sep.join((".signac", "sp_cache.json.gz")),
+    }
+    for src, dst in files_to_move.items():
+        os.rename(
```

Comment:
There are some subtleties about [`os.rename`](https://docs.python.org/3/library/os.html#os.rename) vs. [`os.replace`](https://docs.python.org/3/library/os.html#os.replace) on Windows. I think we always use `os.replace` in signac. I see `os.rename` also comes up elsewhere in this file, so we might want to double-check that globally.
```suggestion
        os.replace(
```

Reply:
Good call, I forgot about this.